From 6d173cf90589af87738018a7e3b3f1883e381bbe Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:42:07 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(91847cf2)..lineage-16.0-android-9.0.0_r31(500faebe) (DO
 NOT SUBMIT)

Bill Yi (3):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (3):
 - Snap for 5044688 from 98cbb36f8de0b3142d13a5c1ea574b97771eb677 to
   pi-qpr2-release
 - Snap for 5101444 from cf8b12a99fc585513a40fd222a422186d8233f66 to
   pi-qpr2-release
 - Snap for 5115288 from 464fb1343365bbcb0f899c94dd7056e8b422b0bc to
   pi-qpr2-release

Change-Id: If158c6cf01851673f827b5cb5fd3f7756f2c8989

Change-Id: I647c27534dac3365f2ebc6e266e6cda52a60ec22
---
 res/values-ar/strings.xml | 2 +-
 res/values-mr/strings.xml | 4 ++--
 res/values-ro/strings.xml | 4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index 44d8e655..48e32f71 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -42,7 +42,7 @@
     <string name="prompt_connect_to_network" msgid="8511683573657516114">"هل تريد الاتصال بالشبكة <xliff:g id="NETWORK_SSID">%1$s</xliff:g>؟"</string>
     <string name="beam_requires_nfc_enabled" msgid="2800366967218600534">"‏يتطلب شعاع Android تفعيل ميزة NFC. هل تريد تفعيلها؟"</string>
     <string name="android_beam" msgid="1666446406999492763">"‏شعاع Android"</string>
-    <string name="beam_requires_external_storage_permission" msgid="8798337545702206901">"لا يحتوي التطبيق على إذن سعة تخزين خارجية، إلا أنه مطلوب لإرسال هذا الملف"</string>
+    <string name="beam_requires_external_storage_permission" msgid="8798337545702206901">"لا يحتوي التطبيق على إذن مساحة تخزين خارجية، إلا أنه مطلوب لإرسال هذا الملف"</string>
     <string name="title_confirm_url_open" msgid="8069968913244794478">"هل تريد فتح الرابط؟"</string>
     <string name="summary_confirm_url_open" product="tablet" msgid="3353502750736192055">"‏تلقى جهازك اللوحي رابطًا عبر ميزة NFC:"</string>
     <string name="summary_confirm_url_open" product="default" msgid="1246398412196449226">"‏تلقى هاتفك رابطًا عبر ميزة NFC:"</string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index de500db4..cbe4417d 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -27,11 +27,11 @@
     <string name="confirm_pairing" msgid="4112568077038265363">"तुम्ही <xliff:g id="DEVICE_NAME">%1$s</xliff:g> ब्लूटूथ डिव्हाइस जोडू इच्छिता याची तुम्हाला खात्री आहे?"</string>
     <string name="pair_yes" msgid="3525614878559994448">"होय"</string>
     <string name="pair_no" msgid="5022308368904055020">"नाही"</string>
-    <string name="tap_again_to_pay" msgid="5754988005412859897">"<xliff:g id="APP">%1$s</xliff:g> सह देय देण्यासाठी पुन्हा टॅप करा"</string>
+    <string name="tap_again_to_pay" msgid="5754988005412859897">"<xliff:g id="APP">%1$s</xliff:g> सह पेमेंट देण्यासाठी पुन्हा टॅप करा"</string>
     <string name="tap_again_to_complete" msgid="5423640945118279123">"<xliff:g id="APP">%1$s</xliff:g> सह पूर्ण करण्यासाठी पुन्हा टॅप करा"</string>
     <string name="transaction_failure" msgid="7828102078637936513">"हा व्यवहार <xliff:g id="APP">%1$s</xliff:g> सह पूर्ण होऊ शकला नाही."</string>
     <string name="could_not_use_app" msgid="8137587876138569083">"<xliff:g id="APP">%1$s</xliff:g> वापरू शकलो नाही."</string>
-    <string name="pay_with" msgid="5531545488795798945">"यासह देय द्या"</string>
+    <string name="pay_with" msgid="5531545488795798945">"यासह पेमेंट करा"</string>
     <string name="complete_with" msgid="6797459104103012992">"यासह पूर्ण करा"</string>
     <string name="default_pay_app_removed" msgid="4108250545457437360">"टॅप आणि पे साठीची तुमची अग्रमानांकित सेवा काढली. दुसरी निवडायची?"</string>
     <string name="ask_nfc_tap" msgid="2925239870458286340">"पूर्ण करण्यासाठी दुसरे डिव्हाइस टॅप करा"</string>
diff --git a/res/values-ro/strings.xml b/res/values-ro/strings.xml
index 61172304..80346199 100644
--- a/res/values-ro/strings.xml
+++ b/res/values-ro/strings.xml
@@ -43,8 +43,8 @@
     <string name="beam_requires_nfc_enabled" msgid="2800366967218600534">"Android Beam necesită activarea NFC. Activați?"</string>
     <string name="android_beam" msgid="1666446406999492763">"Android Beam"</string>
     <string name="beam_requires_external_storage_permission" msgid="8798337545702206901">"Aplicația nu are permisiune de stocare externă. Aceasta este necesară pentru transmiterea fișierului."</string>
-    <string name="title_confirm_url_open" msgid="8069968913244794478">"Deschideți linkul?"</string>
+    <string name="title_confirm_url_open" msgid="8069968913244794478">"Accesați linkul?"</string>
     <string name="summary_confirm_url_open" product="tablet" msgid="3353502750736192055">"Tableta dvs. a primit un link prin NFC:"</string>
     <string name="summary_confirm_url_open" product="default" msgid="1246398412196449226">"Telefonul dvs. a primit un link prin NFC:"</string>
-    <string name="action_confirm_url_open" msgid="3458322738812921189">"Deschideți linkul"</string>
+    <string name="action_confirm_url_open" msgid="3458322738812921189">"Accesați linkul"</string>
 </resources>
-- 
2.17.1

